server/webdriver/shared_tests/scatter_test.py

```diff
@@ -57,6 +60,9 @@ def test_charts_from_url(self):
 
     # Wait until the chart has loaded.
     shared.wait_for_loading(self.driver)
+    WebDriverWait(self.driver, LONG_TIMEOUT).until(shared.charts_rendered)
```

Comment:
how about adding a variant of wait_elem that we use here and elsewhere for things we know are slower? something like wait_elem_slow

Reply:
wait_elem explicitly uses `presence_of_element_located` as an expected condition, so instead of a wait_elem, I added a new `wait_for_charts_to_render`. I noticed the way we currently use shared.charts_rendered has an inefficient double wait, so I've also updated the function to more clearly be an explicit wait condition called by wait_for_charts_to_render.

I also didn't add a wait_elem_slow because I think we can just use the timeout parameter in wait_elem